bridge/client/app/_components/_dialogs/ktb-confirmation-dialog/ktb-confirmation-dialog.component.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { KtbConfirmationDialogComponent } from './ktb-confirmation-dialog.component';
+import { AppModule } from '../../../app.module';
+import { MAT_DIALOG_DATA, MatDialogModule, MatDialogRef } from '@angular/material/dialog';
+
+describe('KtbDeletionDialogComponent', () => {
+  let component: KtbConfirmationDialogComponent;
+  let fixture: ComponentFixture<KtbConfirmationDialogComponent>;
+  const dialogData = {
+    sequence: {
+      shkeptncontext: 'f6a38eb6-e99d-4d14-ab4c-3e94ed288b45',
+      name: 'delivery'
+    },
+    confirmCallback: (params: any) => { }
+  };
+  const dialogRefMock = {
+    close: () => { }
+  };
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      imports: [AppModule, MatDialogModule],
+      providers: [
+        {provide: MAT_DIALOG_DATA, useValue: dialogData},
+        {provide: MatDialogRef, useValue: dialogRefMock}
+      ],
+    }).compileComponents();
+
+    fixture = TestBed.createComponent(KtbConfirmationDialogComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+});
```

Comment:
Add `it should close the dialog` and spy on `confirmCallback` and `dialog.close` that both are called